crypto/modes/cfb128.c

```diff
@@ -92,9 +99,10 @@ void CRYPTO_cfb128_encrypt(const unsigned char *in, unsigned char *out,
                 while (len >= 16) {
                     (*block) (ivec, ivec, key);
                     for (; n < 16; n += sizeof(size_t)) {
-                        size_t t = *(size_t *)(in + n);
-                        *(size_t *)(out + n) = *(size_t *)(ivec + n) ^ t;
-                        *(size_t *)(ivec + n) = t;
+                        size_t t = *(ua_size_t *)(in + n);
```

Comment:
Should this be `ua_size_t` here?

Reply:
No, "t" should be placed in aligned memory, only "in + n" is actually unaligned.